src/main/java/org/activiti/cloud/examples/connectors/ExampleConnector.java

```diff
@@ -31,6 +31,9 @@
     //just a convenience - not recommended in real implementations
     private String var1Copy = "";
 
+    //just a convenience - not recommended in real implementations
+    private String inBoundVariable = "";
```

Comment:
May I suggest to store the whole content of inbound variables? `Map<String, Object>` instead of `String`?